projects/1.16/assets/tinkers-construct/tconstruct/lang/zh_cn.json

```
@@ -1,2517 +1,2517 @@
+    "modifier.tconstruct.slurping.description": "按下交互键时 能引用头盔内的流体",
+    "modifier.tconstruct.aqua_affinity": "水性",
+    "modifier.tconstruct.aqua_affinity.flavor": "与海融为一体！",
+    "modifier.tconstruct.aqua_affinity.description": "在水下挖掘时不再受到挖掘速度影响",
+    "modifier.tconstruct.item_frame": "物品展示框",
+    "modifier.tconstruct.item_frame.flavor": "我能目清一切",
+    "modifier.tconstruct.item_frame.description": "为头盔增加物品栏，从而便于查看。非常适合放置时钟或指南针",
+
+    "_comment": "Armor - chestplate",
+    "modifier.tconstruct.knockback_armor": "击退",
+    "modifier.tconstruct.knockback_armor.flavor": "哇哈哈！",
+    "modifier.tconstruct.knockback_armor.description": "任意武器击打后都能让它们飞的更远！",
+    "modifier.tconstruct.armor_power": "能量",
+    "modifier.tconstruct.armor_power.flavor": "你能感受到流过体内能量么？",
+    "modifier.tconstruct.armor_power.description": "让你的武器更加强大",
+    "modifier.tconstruct.strength": "力量",
+    "modifier.tconstruct.strength.flavor": "你能感受到流过体内力量么？",
+    "modifier.tconstruct.strength.description": "让你的武器更加强大",
+    "modifier.tconstruct.unarmed": "徒手攻击",
+    "modifier.tconstruct.unarmed.flavor": "僧侣之选",
+    "modifier.tconstruct.unarmed.description": "使胸甲强化，从而提升徒手攻击的伤害",
+    "recipe.tconstruct.modifier.unarmed": "需要“无持握”作为前置",
+    "modifier.tconstruct.wings": "羽翼",
+    "modifier.tconstruct.wings.flavor": "天空属于我",
+    "modifier.tconstruct.wings.description": "允许你在半空中跳跃，从而在空中翱翔",
+
+    "_comment": "Armor - leggings",
+    "modifier.tconstruct.speedy": "迅捷",
+    "modifier.tconstruct.speedy.flavor": "跑啊……跑啊……",
+    "modifier.tconstruct.speedy.description": "跑的和兔子一样快",
+    "modifier.tconstruct.leaping": "跳跃",
+    "modifier.tconstruct.leaping.flavor": "一跃千里！",
+    "modifier.tconstruct.leaping.description": "腿部的活塞助力，让你能跳的更高",
+    "modifier.tconstruct.pockets": "行囊",
+    "modifier.tconstruct.pockets.flavor": "真是奇妙的发明！",
+    "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，这些物品在潜行时按下交互键时能够拿取",
+    "modifier.tconstruct.shield_strap": "护带",
+    "modifier.tconstruct.shield_strap.flavor": "保护与攻击的理想之选",
+    "modifier.tconstruct.shield_strap.description": "按下护腿交互键能够交换护副手物品",
+    "modifier.tconstruct.tool_belt": "工具带",
+    "modifier.tconstruct.tool_belt.flavor": "恨不得长出六只手么？",
+    "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
+    "modifier.tconstruct.pocket_chain": "锁链行囊",
+    "modifier.tconstruct.pocket_chain.flavor": "一链连双方",
+    "modifier.tconstruct.pocket_chain.description": "潜行按下交互键时，能够访问工具带或护带所有槽位，还能增加一格槽位",
+    "recipe.tconstruct.modifier.pocket_chain": "“锁链行囊”需要“护带”或“工具带”作为前置",
+
+    "modifier.tconstruct.inventory_cannot_remove": "进行此更改将会删除此工具中存储的所有物品",
+
+    "_comment": "Armor - boots",
+    "modifier.tconstruct.feather_falling": "羽落",
+    "modifier.tconstruct.feather_falling.flavor": "呼呼呼",
+    "modifier.tconstruct.feather_falling.description": "软着陆能减少伤害",
+    "modifier.tconstruct.feather_falling.resistance": "摔落抗性",
+    "modifier.tconstruct.soulspeed": "魂速",

```

```suggestion
    "modifier.tconstruct.soulspeed": "灵魂疾行",
```
原版魔咒名，不过与下方统一或是直接“魂速”都可以